api-report/migration-shim.api.md

```diff
@@ -14,9 +14,10 @@ import { IFluidDataStoreRuntime } from '@fluidframework/datastore-definitions';
 import { IFluidHandle } from '@fluidframework/core-interfaces';
 import { IFluidLoadable } from '@fluidframework/core-interfaces';
 import { IGarbageCollectionData } from '@fluidframework/runtime-definitions';
+import { ISharedTree } from '@fluid-experimental/tree2';
 import { ISummaryTreeWithStats } from '@fluidframework/runtime-definitions';
 import { ITelemetryContext } from '@fluidframework/runtime-definitions';
-import { SharedObject } from '@fluidframework/shared-object-base';
+import { SharedTree } from '@fluid-experimental/tree';
```

Comment:
FWIW - I've been doing `SharedTree as LegacySharedTree` to make it more obvious in code which one we're talking about

Reply:
it seem like a bug that private packages even had an api-report, as there should be no published api

Reply:
I kind of like it as a quick summary of exported API surface even if it never gets published as documentation (especially when reviewing PRs that modify the exported surface).